src/Animal.java

```diff
@@ -0,0 +1,13 @@
+public abstract class Animal implements Feed, Walk, Fly, Swim {
```

Comment:
That would be a some kind of super animal)Okey Let's refactor your code. For Example:
All Fishes can swim . so you can write public abstract class Fish extends Animal implements Swimmable. 
so all Fish child classes will have swim() method.
But not all birds can fly cause we have Penguin  which is also a bird but cannot fly(but he can swim). 
so we can't write public abstract class Bird extends Animal implements Flyable